tests/php-unix.ini

```diff
@@ -1,5 +1,6 @@
 [PHP]
 ;extension_dir = "./ext"
+extension=tokenizer.so
```

Comment:
Without this I get a lot of errors: Fatal error: Call to undefined function Nette\DI\token_get_all() in ...nette/di/src/DI/PhpReflection.php on line 117